cluster/addons/fluentd-elasticsearch/es-image/Makefile

```diff
@@ -16,7 +16,7 @@
 
 # The current value of the tag to be used for building and
 # pushing an image to gcr.io
-TAG = v5.4.0-1
+TAG = v5.5.1
```

Comment:
@aknuds1 How about using the PREFIX, NAME, TAG notation as in `cluster/addons/fluentd-elasticsearch/kibana-image/Makefile` for consistency?